src/05-objects.js

```diff
@@ -118,7 +154,15 @@ function addKeyToObject(object, key) {}
  *  addKeyValuePairToObject({ up: true }, "left", false);
  *  //> { up: true, left: false }
  */
-function addKeyValuePairToObject(object, key, val) {}
+function addKeyValuePairToObject(object, key, val) {
+  newObject = {};
+  newObject[key] = val;
+  //newObject[val] = val;
```

Comment:
We should start getting into the habit of removing and ` console.log() ` statements or commented out code from our files before we push.